open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/util/ConsentManageUtil.java

```diff
@@ -621,4 +640,63 @@ public static boolean isConsentExpirationTimeValid(String expDateVal) {
         }
     }
 
+    /**
```

Comment:
Shall we remove commented code if this is not required?

Reply:
Fixed in [876c68f]